frontend/script.js

```diff
@@ -0,0 +1,57 @@
+function showSideBar() {
```

Comment:
Similar to how it was done for the css files (adding the `orders101-ui.css` "wrapper"), could we name this file `main.js` or similar, separate the content which is currently in it, and then import them into this file as a wrapper?